### DISCUSSION TITLE
Default Username & Password CouchDB

### DISCUSSION THREAD

**Discussion by muksidin** [Budibase/budibase#8449] [Help] [answered]
Hello, what is default username & password couchdb, i install budibase as self hosted & using Docker Hub.
Thanks

**Comment by andz-bb** [comment, marked as answer]
Hey @muksidin, you should be able to see your couchdb credentials in your `docker-compose.yaml` file. Look for `COUCHDB_USER` and `COUCHDB_PASSWORD` in your environment variables. On my instance the username is `admin` and the password is a randomly generated string. Hope this helps!

**Reply by muksidin** [reply]
Yes, i think your answer if i install budibase using docker compose. but i install it using single image docker.

**Reply by andz-bb** [reply]
@muksidin hey, with the single image setup there should be a mounted volume which contains a `.env` file - this will contain those CouchDB details for your instance.

**Reply by muksidin** [reply]
@andz-bb owh, can you help me how to access `.env` file? i am not familiar in docker.

**Reply by andz-bb** [reply]
@muksidin I was using docker desktop and was able to look in my volumes for the one used by the budibase container. Mine is called `code_budibase_data` for my single image instance. Within that volume the `.env` file can be accessed.

<img width="1270" alt="image" src="https://user-images.githubusercontent.com/110921612/199560213-a9cd1505-58c9-4961-8194-7676e3ad7de3.png">


If you only have access to a cli I think you can use `docker volume ls` to list the volumes and then `docker volume inspect your_volume_name` to find the path to the files on it.

**Reply by muksidin** [reply]
@andz-bb great,thanks alot for your help :)

**Comment by enthus1ast** [comment]
I did it with:

```
$ docker ps 

CONTAINER ID   IMAGE                      COMMAND                  CREATED        STATUS                 PORTS                                                                                      NAMES
c058748149d9   budibase/budibase:latest   "tini -- /docker-ent…"   2 weeks ago    Up 2 hours (healthy)   443/tcp, 2222/tcp, 4369/tcp, 5984/tcp, 9100/tcp, 0.0.0.0:10000->80/tcp, :::10000->80/tcp   budibase


$ docker exec -it c058748149d9 bash    # change this to the id of the container
# cd /data/
# cat .env 
COUCHDB_USER=admin
COUCHDB_PASSWORD=admin
[....]
```

so it seems the default username and password is:

Url: http://127.0.0.1:10000/db/_utils/#
User: admin
Pass: admin
